packages/pymedphys_pinnacle/src/pymedphys_pinnacle/export/pinnacle_plan.py

```diff
@@ -125,13 +125,7 @@ def trials(self):
             if isinstance(self._trials, dict):
                 self._trials = [self._trials['Trial']]
 
-            # Select trial with FINAL in name as active trial for this plan
-            # If no FINAL trial found then select first trial
-            for trial in self._trials:
-                if 'FINAL' in trial['Name'].upper():
```

Comment:
Did you want this to be accessible via a parameter? It's quite important that you ["Eat your own dog food"](https://en.wikipedia.org/wiki/Eating_your_own_dog_food), so ideally the code you write here can be the code you end up using internally. Therefore, you would want it to be able to support your internal processes.

Reply:
What I mean is have a parameter like `trial_name_selector` which defaults to `None` but allows you to adjust it to something like `FINAL`

Reply:
Perhaps in a future version, but I will script that functionality from where I call on the API and I would encourage others to do the same.

Reply:
Makes sense